test/runtime/specs/juttle-spec/expressions/function-call.spec.md

```diff
@@ -76,8 +76,8 @@ Produces an error when passed too many parameters
 
   * Error: function f expects 1 to 2 arguments but was called with 3
 
-Returns correct result when passed required parameters (module)
----------------------------------------------------------------
+Returns correct result when passed required parameters (module) (with optional parameters)
```

Comment:
I’d just do `(module, with optional parameters)`. We do it like this elsewhere IIRC.